coredump_uploader/__init__.py

```diff
@@ -488,6 +490,7 @@ def upload(self, path_to_core):
             ["uname", "-s", "-r"], stdout=subprocess.PIPE, stdin=subprocess.PIPE,
```

Comment:
@drindhauser entirely unrelated to this PR.  I'm curious however as to why invoking `uname` as a subprocess was chosen over using `os.uname()`?  Are there cases where they are different?